fix(collaboration): fixed some minor things (#11035)
newrelic/docs-website: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx

* **Create shared knowledge and history.** Collaborating within the New Relic platform allows your team to build a searchable body of shared knowledge.
## Comment on any page [#comment-on-any-page]

> On the global navigation, there's an “add comment” icon on most pages. A green dot on the icon means there's an open discussion on the page, while a gray dot indicates a closed discussion that may have information to help resolve a current issue. 

Reviewer: Per style guide, we bold UI elements and use the verbiage as it appears in the UI. So this should be stylized as **Add comment** iirc.

## At-mention your teammates and notify by email [#at-mention-your-teammates]

> Like other collaboration tools, when you at-mention (@) your teammates in a comment, they'll be notified by email. 

Reviewer: ```suggestion
Like other collaboration tools, when you at-mention (@) your teammates in a comment, they'll receive an email notification. 
```

taking out passive voice :)

## Built-in screenshot sharing [#built-in-screenshots]

> Use our integrated screenshot selection tool you can attach screenshots to your comments. Just click the “screenshot” icon, select the portion of the screen you want, and submit your comment. 

Reviewer: ```suggestion
Attach screenshots to your comments by clicking our integrated screenshot selection tool. Just click the “screenshot” icon, select the portion of the screen you want, and submit your comment. 
```

clicking or using for the verb. But makes this more action-specific and frontloads what outcome the user may want!
